.busted

```diff
@@ -2,17 +2,19 @@ return {
   _all = {
     coverage = true,
     shuffle = true,
-    lpath = "./src/?.lua;./spec/?.lua"
+    recursive = true,
+    verbose = true,
+    lazy = false,
+    ["auto-insulate"] = true,
+    ["keep-going"] = true,
+    ["defer-print"] = false,
+    lpath = "./src/?.lua;./spec/?.lua",
+    ROOT = {"spec/"},
```

Comment:
Please explain to me what the "busted" file means.

Reply:
Busted is a test runner for Lua, this is it's config file.